src/emulator/load_self.cpp

```diff
@@ -1,160 +0,0 @@
-// Vita3K emulator project
```

Comment:
Moving this file, and making changes to it should be in separate commits IMO, it's impossible to see what you changed this way.